pkg/apiclient/rollout/rollout.proto

```diff
@@ -0,0 +1,179 @@
+syntax = "proto3";
+option go_package = "github.com/argoproj/argo-rollouts/pkg/apiclient/rollout";
+
+import "google/api/annotations.proto";
+import "github.com/argoproj/argo-rollouts/pkg/apis/rollouts/v1alpha1/generated.proto";
+import "k8s.io/apimachinery/pkg/apis/meta/v1/generated.proto";
+import "google/protobuf/empty.proto";
+
+package rollout;
+
+message RolloutQuery {
+    string name = 1;
+}
+
+message SetImageQuery {
```

Comment:
Since this is an action and not a get, I think this should be named `SetImageRequest` instead of `SetImageQuery`